src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs

```diff
@@ -5457,6 +5457,263 @@ public class B : A
             await TestInRegularAndScriptAsync(testText, expected);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsPullMemberUp)]
+        public Task TestRefactoringFieldFromModifier()
```

Comment:
why not have the feature do this work?  i don't love that the helper service is cahnging in this regard.